src/DynamoCoreWpf/Views/Gallery/GalleryView.xaml

```diff
@@ -0,0 +1,135 @@
+﻿<UserControl x:Class="Dynamo.Wpf.Views.Gallery.GalleryView"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+        xmlns:ui="clr-namespace:Dynamo.UI"
+        xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
+        xmlns:viewModels="clr-namespace:Dynamo.ViewModels"
+        d:DesignHeight="600" 
+        d:DesignWidth="600"
+        VerticalAlignment="Center"
+        HorizontalAlignment="Center"
+        Background="White"
+        MouseLeftButtonDown="GalleryView_MouseLeftButtonDown"
+        mc:Ignorable="d"
+        Focusable="True"
+        Padding="0,0,0,20">
+    
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <UserControl.InputBindings>
+        <KeyBinding Key="Escape"             
+                    Command="{Binding Path=CloseGalleryCommand}" />
+    </UserControl.InputBindings>
+    
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="100"></ColumnDefinition>
+            <ColumnDefinition Width="*"></ColumnDefinition>
+            <ColumnDefinition Width="100"></ColumnDefinition>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="50"></RowDefinition>
```

Comment:
@Benglin, the version number and the close button are on that row. I don't know how we can put those things using margin.